src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -904,8 +904,13 @@ function GlobeControls(view, target, radius, options = {}) {
 
             event.preventDefault();
 
+            const staticPos = window.getComputedStyle(event.target.parentElement).position !== 'static';
```

Comment:
could you factorize this part that appears several times?

Reply:
Nope, it depends on the `event`, and this variable is available in the function scope only. As `GlobeControls` is a sort of singleton (correct me if I'm wrong), we can't compute the value at the init of the class.

Reply:
Ok, you could pass the event as a parameter.
In short, it isn't important.